specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-11-15-preview/cosmos-db.json

```diff
@@ -9422,6 +9426,24 @@
         "encryptionAlgorithm"
       ]
     },
+    "MaterializedViewDefinition": {
+      "type": "object",
+      "description": "Materialized View definition for the container.",
+      "properties": {
+        "sourceCollectionId": {
+          "description": "Name of the Cosmos DB SQL source container",
+          "type": "string"
+        },
+        "definition": {
+          "description": "Definition for the SQL Materialized view",
```

Comment:
Modify to "SQL query used to fetch data from source table to populate the SQL Materialized view" or in similar words. Definition may not give clear idea to end user on purpose of field.